nixos/modules/security/acme.nix

```diff
@@ -136,6 +136,23 @@ let
           challenge to ensure the DNS entries required are available.
         '';
       };
+
+      mustStaple = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Turns on the OCSP Must-Staple TLS extension.
+          Make sure you know what you're doing!
```

Comment:
I think a statement such as this deserves a link with an explanation of the situation.

Reply:
Resolved in #80900.